src/python/sheet_parameters.py

```diff
@@ -50,4 +50,29 @@
         "required_columns": ("process", "key"),
         "unique_columns": ("process", "key"),
     },
+    {
+        "name": "computation",
+        "required_columns": ("experiment", "name", "type"),
+        "unique_columns": ("name"),
+    },
+    {
+        "name": "prerequisite computation",
+        "required_columns": ("computation", "prerequisite"),
+        "unique_columns": ("computation", "prerequisite"),
+    },
+    {
+        "name": "computational process",
+        "required_columns": ("experiment", "name", "type"),
+        "unique_columns": ("name"),
+    },
+    {
+        "name": "software configuration",
+        "required_columns": ("name", "version"),
+        "unique_columns": ("name"),
+    },
+    {
+        "name": "input & output data",
+        "required_columns": ("computation or computational process", "input data"),
+        "unique_columns": ("computation or computational process", "input data"),
+    },
```

Comment:
I really like the way that you have `sheet_parameters.py` file because that makes it super clean, readable, and easy to maintain. Well done! 🎉 🏅